src/main/resources/db/migration/V40.0__DDL_update_pto_table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE pto ALTER COLUMN days TYPE decimal(2,1) USING days::decimal;
```

Comment:
let's change to 
`ALTER TABLE pto ALTER COLUMN days TYPE decimal(16,2) USING days::decimal;`

16 is the entire part need to map from the current int. Also it's common to use 2 as precision